build.gradle

```
@@ -18,6 +18,9 @@ dependencies {
 	// spring
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+    implementation 'org.springframework.boot:spring-boot-starter-data-redis'
```

```suggestion
    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
    implementation 'org.springframework.boot:spring-boot-starter-data-redis'
```

기존 코드는 들여쓰기를 tab으로 하고 있고, 원철님은 space로 추가해주셨는데 통일을 해보면 어떨까요?